src/main/resources/assets/js/app/highchart.es6

```diff
@@ -119,20 +119,18 @@ export function init() {
           },
         }
 
-        // Only show plotOption marker on last data element
-        if (canvas.data('type') === 'line') {
-          config.series.forEach(function (series) {
-            const lastIndex = series.data.length - 1
-            series.data.forEach(function (data, index) {
-              series.data[index] = {
-                y: parseFloat(data),
-                marker: {
-                  enabled: index === lastIndex,
-                },
-              }
-            })
-          })
-        }
+        // Only show plotOption marker on last data element / single data point series
+        config.series.forEach((series) => {
+          const isLineSeries = canvas.data('type') === 'line'
+          const nonNullPoints = series.data.filter((data) => data !== null).length
+
+          series.data = series.data.map((data, index) => ({
+            y: parseFloat(data),
+            marker: {
+              enabled: isLineSeries && (nonNullPoints === 1 || index === series.data.length - 1),
```

Comment:
To get mark on last point on every line, try 
`enabled: index === nonNullPoints - 1,`

Reply:
index === nonNullPoints - 1

Reply:
Enabled: nonNullPoints === 1 || index === nonNullPoints - 1, 

nonNullPoints === 1, is necessary to make this work correctly as the index of the series with one point could be at any position in the dataset table, thus without nonNullPoints === 1 this fix only works for points at index nonNullPoints -1